inspirehep/config.py

```diff
@@ -69,11 +69,11 @@ def _(x):
 
 # Celery
 # ======
-BROKER_URL = "amqp://guest:guest@localhost:5672//"
-CELERY_RESULT_BACKEND = "amqp://guest:guest@localhost:5672//"
+CELERY_BROKER_URL = "pyamqp://guest:guest@localhost:5672//"
+CELERY_RESULT_BACKEND = "rpc://guest:guest@localhost:5672//"
 CELERY_ACCEPT_CONTENT = ['json', 'msgpack', 'yaml']
```

Comment:
But, why not moving to `msgpack` by default and see if we pass? That would give quite a boost in performance, for small tasks.